gulpfile.js

```diff
@@ -460,6 +528,8 @@ gulp.task('release', ['apps', 'clean-release'], function () {
     if (platforms.indexOf('win64') !== -1) {
         releaseWin('win64');
```

Comment:
Also out of curiosity. Why not `merge.add`? When building releases for multi platform at once, it should also be possible to build for osx, windows and/or linux.

Reply:
The `makensis` command does not work with async streams. Is a sync command, I think, but I will try to verify it, maybe I'm wrong ;)

Reply:
I have seen than the sync command is other, so this is async, but I'm not too sure if this will return an stream. I will made some tests.